forum/qa-plugin/tips-widget/tips-widget-page.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+class tips_widget_page
+{
+	function match_request($request)
+	{
+		if ($request == 'tips')
+			return true;
+		
+		return false;
+	}
+	
+	function suggest_requests()
+	{
+		return [['title'   => 'Porady nie od parady',
+		          'request' => 'tips',
+		          'nav'     => null]];
+	}
+	
+	function process_request($request)
+	{
+		if (!qa_opt('tips-enable'))
+		{
+			return include QA_INCLUDE_DIR.'qa-page-not-found.php';
```

Comment:
to jest niedorzeczność

Reply:
Czemu? Zapis w pełni działa, zwracając stronę z domyślnym zapisem "Nie znaleziono strony". Jeśli masz lepszy sposób na rozwiązanie tego, to z miłą chęcią go poznam.